nautobot/docs/user-guide/administration/installation/nautobot.md

```diff
@@ -266,6 +266,45 @@ Nautobot does not come with any predefined user accounts. You'll need to create
 nautobot-server createsuperuser
 ```
 
+## Build Nautobot 2.0 UI
+
+Nautobot 2.0 introduces a new user interface. This interface is built using [React](https://react.dev/) and requires Node.js version 18 or higher to build the UI. 
+
+### Install Node.js
+
+Node.js version 18 is only listed on package manager repositories for Debian 12 Bookworm, Ubuntu 22.10 kinetic and RHEL 8.
+
+Debian Bookworm or Ubuntu kinetic or higher:
+
+```no-highlight
+sudo apt-get -y install npm
+```
+
+RHEL 8/9:
+
+```no-highlight
+sudo dnf module install nodejs:18/common
+```
```

Comment:
Should these be included or referenced directly in `centos.md` and `ubuntu.md` instead of here?